packages/@aws-cdk/aws-lambda/package.json

```diff
@@ -156,6 +156,7 @@
       "docs-public-apis:@aws-cdk/aws-lambda.Runtime.PYTHON_3_7",
       "docs-public-apis:@aws-cdk/aws-lambda.Runtime.PYTHON_3_8",
       "docs-public-apis:@aws-cdk/aws-lambda.Runtime.RUBY_2_5",
+      "docs-public-apis:@aws-cdk/aws-lambda.Runtime.RUBY_2_7",
```

Comment:
Would be better to document it instead of adding more and more exceptions over time 🤷🏻‍♂️

Reply:
The docs for the Runtime construct are automatically generated here: https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-lambda.Runtime.html 

Perhaps I could link this page from the main Lambda README.md, but that wouldn't solve adding exceptions every time. Any suggestions perhaps? :) @RomainMuller

Reply:
It's just about adding a `/** */` comment to the `Runtime` constants. The `docs-public-apis` rule basically requires TSDoc comments on all public APIs. The cost of adding those here is pretty low (because these will be boring - e.g: "The Ruby 2.7 runtime (`ruby2.7`)", but will make the doc experience ever so slightly nicer).